nucypher/blockchain/eth/sol/source/contracts/lib/UmbralDeserializer.sol

```diff
@@ -76,7 +95,7 @@ library UmbralDeserializer {
         _pointer = copyPoint(_pointer, proof.pointKFragCommitment);
         _pointer = copyPoint(_pointer, proof.pointKFragPok);
         proof.bnSig = uint256(getBytes32(_pointer));
-        _pointer += BIG_NUM_SIZE;
+        _pointer += BIGNUM_SIZE;
```

Comment:
No bounds checking for `_pointer`?

Reply:
this line `require(_proofBytesLength >= CORRECTNESS_PROOF_SIZE);` checks